### PR TITLE
Added blog post for v0.15.1-alpha

### PR DESCRIPTION
**Pull request by karifabri**

### CODE REVIEW

File: content/posts/releases/v0.15.1-alpha.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+date: 2024-07-08
+title: 'Spice v0.15.1-alpha (July 8, 2024)'
+type: blog
+author: Spice AI ([@spice_ai](https://twitter.com/spice_ai))
+categories: [release]
+tags: [stability, postgressql, duckdb, stream processing, delta lake, data connector, databricks, odbc, graphql, deletion vectors, unnesting]
```
Comment (lukekim) [inline content/posts/releases/v0.15.1-alpha.md]:
```suggestion
tags: [postgres, duckdb, delta lake, databricks, odbc, graphql]
```

File: content/posts/releases/v0.15.1-alpha.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+**GraphQL Object Unnesting:** The [GraphQL Data Connector](https://docs.spiceai.org/data-connectors/graphql) can automatically unnest objects from GraphQL queries using the `unnest_depth` parameter.
+
+## Breaking Changes
+
+None.
+
+## New Contributors
+
+None.
+
```
Comment (lukekim) [inline content/posts/releases/v0.15.1-alpha.md:29]: If there are no new contributors, we should skip this section.

```suggestion
```

Comment (karifabri) [reply content/posts/releases/v0.15.1-alpha.md:29]: Ok, thank you.

File: content/posts/releases/v0.15.1-alpha.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+- [@sgrebnov](https://github.com/sgrebnov)
+- [@ewgenius](https://github.com/ewgenius)
+- [@peasee](https://github.com/peasee)
+- [@lukekim](https://github.com/lukekim)
+
+## What's Changed
+
+### Dependencies
+
+The MySQL, PostgreSQL, SQLite and DuckDB DataFusion TableProviders developed by Spice AI have been donated to the [datafusion-contrib/datafusion-table-providers](https://github.com/datafusion-contrib/datafusion-table-providers) community repository.
```
Comment (lukekim) [inline content/posts/releases/v0.15.1-alpha.md:45]:
```suggestion
The MySQL, PostgreSQL, SQLite and DuckDB DataFusion TableProviders developed by Spice AI have been donated to the [datafusion-contrib/datafusion-table-providers](https://github.com/datafusion-contrib/datafusion-table-providers) community repository.

From the v0.15.1-alpha release, a new dependency is taken on `datafusion-contrib/datafusion-table-providers`
```

**Review by lukekim — APPROVED** [review]

**Review by phillipleblanc — APPROVED** [review]

**Pull request merged**